controlplane/src/core/repositories/BillingRepository.ts

```diff
@@ -23,12 +24,11 @@ export class BillingRepository {
     });
   }
 
-  private createOrRetrieveCustomerId = async ({ id, email }: { id: string; email?: string }) => {
+  private createStripeCustomer = async ({ id, email }: { id: string; email?: string }) => {
```

Comment:
createOrRetrieveCustomerId is more descriptive/clear what this method does, it will actually upsert and return the id.

Reply:
so `upsertStripeCustomerId` ?

Reply:
Done